deps/v8/third_party/ittapi/LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2019 Intel Corporation. All rights reserved.
```

Comment:
This file isn't part of the ittapi repo and will be deleted next time we update V8. Shouldn't we read the license from [LICENSES/BSD-3-Clause.txt](https://github.com/intel/ittapi/blob/a3911fff01a775023a06af8754f9ec1e5977dd97/LICENSES/BSD-3-Clause.txt) instead?